.github/workflows/screenshots.yml

```diff
@@ -0,0 +1,79 @@
+name: Screenshots
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main    
+
+jobs:
+  screenshots:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
A lot of this code is duplicated between several workflow files, which is a pain. Unfortunately GitHub Actions doesn't provide a good way to share these steps between multiple workflows yet. See https://github.com/actions/runner/issues/646 for more details.

Reply:
If we wrote an action to combine these steps would that work?

Reply:
There is a thing called a [composite action](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-composite-run-steps-action), but it won't allow you to have the `uses` syntax (and reference other actions)